circleci/docker-publish

```diff
@@ -30,12 +30,22 @@ check_ecr_vars() {
   if [[ -z $ECR_PUSH_SECRET ]]; then echo "Missing var for ECR: ECR_PUSH_SECRET" && exit 1; fi
 }
 
+install_awscli(){
+  # awscli required for ECR publish steps
```

Comment:
Nit. To help with debuggability, added `echo` that download is about to happen

Reply:
Added log line here and updated other log lines to match format already in use in this file, in f2f2d1196e694e985812c0e5b85b406665f11db6.